app/controllers/api/subcollections/alert_definitions.rb

```diff
@@ -0,0 +1,10 @@
+module Api
+  module Subcollections
+    module AlertDefinitions
+      def alert_definitions_query_resource(object)
+        return {} unless object.respond_to?(:miq_alerts)
+        object.miq_alerts
```

Comment:
That would give us a MiqPolicySet's MiqPolicies but will not give us a VM MiqPolicies.
Is that intentional?

If you need that you should call get_policies (see miq_policy_mixin.rb)